### ISSUE TITLE
cursor/input gets messy on WSL v1 (Ubuntu), with/without ConPTY

### ISSUE THREAD

**Issue by koreno** [Eugeny/tabby#5273]
Version: 1.0.168
Platform: Windows x64 10.0.19044
Plugins: search-in-browser, clickable-ips
Frontend: xterm-webgl

![tabby-cursor](https://user-images.githubusercontent.com/180551/147248309-0ba7cfca-4e19-485c-81a7-a94cbecfe47c.gif)

**Comment by Eugeny** [comment]
Do you mean the ghostly outlines in the background? These are often caused by misbehaving GPU drivers, an easy workaround is to tick `Disable GPU acceleration` in Settings -> Window (v169), or set Tabby to run on the internal GPU in Windows settings (if you have two).

**Comment by koreno** [comment]
No, not ghostly outlines. This not a rendering issue, it's the absolute position of the cursor and the edit point. look closely at where the cursor is at the start of the line (right after the >>), but then the first character appears after a space. Then the character appears on-top of the typed characters, instead of ahead of it.

**Comment by Eugeny** [comment]
Oh I get it - it's likely due to that lightning bolt emoji incorrectly having double width -  please grab a dump of your terminal state as described here: https://github.com/Eugeny/terminus/wiki/Posting-terminal-state

**Comment by koreno** [comment]
Here's another mess:

![tabby-cursor2](https://user-images.githubusercontent.com/180551/147292705-1f726475-31ea-4293-9087-ffd7c117e491.gif)

**Comment by koreno** [comment]
```"\u001b[32mofer.koren@Oferk-win\u001b[0m /mnt/c/Users/ofer.koren \u001b[31m»\u001b[0m ./comet.sh --remote git:l23cr.vstd.int:/vast/orion run --without-jira-test-report --skip-harvest --skip-ensure-online pysrc/tests -k test_alarm_forbidden_and_not_raised -b pysrc/tests/comet/alarms.py:35 --skip-switches --skip-ensure-online\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m /mnt/c/Users/ofer.koren \u001b[31m»\u001b[0m                                                                                                                                                                                                                        \u001b[31m130 ↵\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m /mnt/c/Users/ofer.koren \u001b[31m»\u001b[0m                                                                                                                                                                                                                        \u001b[31m130 ↵\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m /mnt/c/Users/ofer.koren \u001b[31m»\u001b[0m cd ~/Sources/orion                                                                                                                                                                                                     \u001b[31m130 ↵\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m ~/Sources/orion \u001b[32m±\u001b[33mcomet-bigdata-4.3\u001b[0m \u001b[31m»\u001b[0m gs\r\nOn branch comet-bigdata-4.3\r\nnothing to commit, working tree clean\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m ~/Sources/orion \u001b[32m±\u001b[33mcomet-bigdata-4.3\u001b[0m \u001b[31m»\u001b[0m cd ignore/A\r\n\u001b[32mofer.koren@Oferk-win\u001b[0m ~/Sources/orion/ignore/A \u001b[32m±\u001b[33mTEAM/infra⚡\u001b[0m \u001b[31m»\u001b[0m ./com"```

**Comment by koreno** [comment]
Indeed it's only there when the emoji is there... that's from an oh-my-zsh theme.
